Task_1/setup.py

```diff
@@ -28,11 +28,11 @@
     ],
     include_package_data=True,
     install_requires=[
-        'openfl @ git+https://github.com/intel/openfl.git@f4b28d710e2be31cdfa7487fdb4e8cb3a1387a5f',
-        'GANDLF @ git+https://github.com/CBICA/GaNDLF.git@e4d0d4bfdf4076130817001a98dfb90189956278',
+        'openfl @ git+https://github.com/securefederatedai/openfl.git@kta-intel/fets-2024-patch-1',
```

Comment:
This seems to be problematic:

![image](https://github.com/user-attachments/assets/eaceab9d-f15d-4b9f-ae5b-2244ea34c1d8)

Is there any plan to merge this with openfl's `develop` branch so that we can use that instead? This seems prone to breaking, right? Tagging @kta-intel for guidance.

Reply:
I just opened a PR for it: https://github.com/securefederatedai/openfl/pull/1141
Once it is merged, we can delete that branch and update the `setup.py` to use the `develop` branch directly

Couple notes:
1. I moved the conditional into `loader_gandlf.py` instead of `runner_gandalf.py` since it is logically a better spot to check if we are running in inference mode, rather than setting another. Simple move that tests fine on my end, but be mindful that it doesn't affect anything else on your end
2. The native API (i.e. `import openfl.native as fx`) will likely be deprecated at some point in this or an upcoming release, so it might be a good idea to pin to this specific openfl commit rather than `develop` as a whole

Reply:
It has been merged

Point 2 is still being discussed. Considering the importance of the FeTS challenge, let me open up the discussion and propose a way to preserve it at least for v1.7

Reply:
> propose a way to preserve it at least for v1.7

This would be the ideal solution. And even until **at least** 1.9 (considering the conventions of semantic versioning).